lib/LeroyMerlin/ruleset.xml

```diff
@@ -130,8 +130,8 @@
     <rule ref="SlevomatCodingStandard.ControlStructures.RequireShortTernaryOperator"/>
     <!-- Require usage of null coalesce operator when possible -->
     <rule ref="SlevomatCodingStandard.ControlStructures.RequireNullCoalesceOperator"/>
-    <!-- Force yoda conditions -->
-    <rule ref="SlevomatCodingStandard.ControlStructures.RequireYodaComparison"/>
+    <!-- Disallow yoda conditions -->
+    <rule ref="SlevomatCodingStandard.ControlStructures.DisallowYodaComparison"/>
```

Comment:
What about just remove this line and dont check this rule at all? its really very rare the case where this may cause a problem and, for me, is just a standard.

Reply:
What kind of problem, could you explain better?

Reply:
A problem when you trying to compare and end up assign a value to a variable.
e.g `if ($count = 2) { `...

But the problem is not the point, the question is: We really need to check if its yoda or not? instead of force the opposite of yoda, just dont check at all.

Reply:
> We really need to check if its yoda or not?

Yes, we could. It's one of the initial proposals. I'll make clear the voting options and what we voted so far.